test/crypto/sign.js

```diff
@@ -50,28 +46,39 @@ describe('sign', () => {
 
 	describe('#signMessageWithSecret', () => {
 		const signedMessage = signMessageWithSecret(notSecretMessage, defaultSecret);
-
 		it('should signTransaction the message correctly', () => {
-			(signedMessage).should.be.equal(defaultSignature);
+			(signedMessage).should.have.property('message').be.equal(notSecretMessage);
+			(signedMessage).should.have.property('signature').be.equal(defaultSignature);
+			(signedMessage).should.have.property('publicKey').be.equal(defaultPublicKey);
```

Comment:
Either put these in separate tests or assert `eql`ity with an object (to exclude extraneous properties).